test/In.ProjectEKA.HipServiceTest/Consent/Builder/TestBuilder.cs

```diff
@@ -0,0 +1,35 @@
+namespace In.ProjectEKA.HipServiceTest.Consent.Builder
+{
+    using Bogus;
+    using Common.Builder;
+    using HipService.Common.Model;
+    using HipService.Consent;
+    using HipService.Consent.Model;
+
+    public class TestBuilder
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4999696)